packages/ui-spinner/src/Spinner/index.tsx

```diff
@@ -55,6 +55,7 @@ class Spinner extends Component<SpinnerProps> {
   }
 
   ref: Element | null = null
+  titleId: string | undefined
```

Comment:
We decided to mark these as "private" props (not the `ref`, because it has to be accessible from the outside, but the other props that we use internally):

```js
ref: Element | null = null

private titleId: string | undefined
```

Also, this prop in particular is only set in the constructor, so it can be "readonly" too:

```js
private readonly titleId: string | undefined
```

And just one more thing: the way you wrote it is fine, just wanted to show that there is another way to mark it as "possibly undefined" or "optional" with the `?` mark.

```js
private readonly titleId?: string
```